redbot/core/core_commands.py

```diff
@@ -558,14 +558,10 @@ async def info(self, ctx: commands.Context):
     @commands.command()
     async def uptime(self, ctx: commands.Context):
         """Shows [botname]'s uptime."""
-        since = ctx.bot.uptime.strftime("%Y-%m-%d %H:%M:%S")
         delta = datetime.datetime.utcnow() - self.bot.uptime
-        uptime_str = humanize_timedelta(timedelta=delta) or _("Less than one second")
-        await ctx.send(
-            _("Been up for: **{time_quantity}** (since {timestamp} UTC)").format(
-                time_quantity=uptime_str, timestamp=since
-            )
-        )
+        uptime = self.bot.uptime.replace(tzinfo=datetime.timezone.utc)
+        uptime_str = humanize_timedelta(timedelta=delta) or _("Less than one second.")
+        await ctx.send(_(f"Been up for: **{uptime_str}** (since <t:{int(uptime.timestamp())}:f>)"))
```

Comment:
f-strings do not work with i18n, use str.format() instead:
```suggestion
        await ctx.send(
            _(f"Been up for: **{time_quantity}** (since {timestamp})").format(
                time_quantity=uptime_str, timestamp=f"<t:{int(uptime.timestamp())}:f>"
            )
        )
```

Reply:
Didn't know that... fixed it.